internal/config/config.go

```diff
@@ -0,0 +1,145 @@
+package config
+
+import (
+	"io"
+	"os"
+
+	"github.com/BurntSushi/toml"
+	"github.com/fatih/color"
+	"gopkg.in/go-playground/validator.v9"
+)
+
+type tomlConfig struct {
```

Comment:
I'm not really a fan of this. Shiori is intended to be a single file app with zero configuration to make it easy to install and distribute. Hence why it uses Go and SQLite by default.

However, in some case configuration file is a necessity, just like in this LDAP case. With that said, IMHO it would be better if the config file only contained the LDAP part.